src/main/java/nl/tudelft/pixelperfect/event/EventFactory.java

```diff
@@ -6,51 +6,56 @@
 
 /**
  * Not a factory as the design pattern, but used for id's.
- *
+ * <p>
  * Create a random event, with unique id's.
  */
 public class EventFactory {
 
-    private int id;
+  private int id;
 
-    /**
-     * Constructor for EventFactory.
-     */
-    public EventFactory() {
-        id = 0;
-    }
+  /**
+   * Constructor for EventFactory.
+   */
+  public EventFactory() {
+    id = 0;
+  }
 
   /**
    * Generate a random event.
    *
    * @return an Event.
    */
   public Event randomEvent() {
-        int x = (int) Math.floor(Math.random()) % 4;
-        return create(x);
-    }
+    int x = (int) Math.floor(Math.random() * 100) % 4;
+    return create(x);
```

Comment:
I suggest doing (int) Math.random() * numberOfEvents instead of what you have now:
- the casting to int will floor the double.
- modulo 4 is not needed here?